presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/ColumnMapping.java

```diff
@@ -23,10 +25,38 @@
 
 public final class ColumnMapping
 {
-    public static final PredicatePushdownController FULL_PUSHDOWN = domain -> new DomainPushdownResult(domain, Domain.all(domain.getType()));
-    public static final PredicatePushdownController PUSHDOWN_AND_KEEP = domain -> new DomainPushdownResult(domain, domain);
+    public static final int DEFAULT_DOMAIN_SIZE_THRESHOLD = 10_000;
```

Comment:
Should this be a configurable similar to `hive.domain-compaction-threshold` ?

Reply:
We already have fixed limits in Oracle and Sql server, therefore I'm not sure it should be configurable.

Reply:
Added feature/session property